apps/explorer/src/components/Layout/PageLayout.tsx

```diff
@@ -13,8 +13,13 @@ import { Banner } from '~/ui/Banner';
 import { Network } from '~/utils/api/DefaultRpcClient';
 
 export type PageLayoutProps = {
-	gradientContent?: ReactNode;
+	gradientContent?: {
```

Comment:
I would flatten these at top level, but if not maybe we could change this to just `gradient`?